### PR TITLE
documentation update for cms specific options to category

### PR DESCRIPTION
**Pull request by tyfarnan**

### CODE REVIEW

File: docs/source/profiler.rst [hunk 1]
```diff
@@ -761,6 +761,13 @@ Below is an breakdown of all the options.
       * top_k_categories - (int) Number of categories to be displayed when reporting
       * max_sample_size_to_check_stop_condition - (int) The maximum sample size before categorical stop conditions are checked
       * stop_condition_unique_value_ratio - (float) The highest ratio of unique values to dataset size that is to be considered a categorical type
+      * cms - (Boolean) Enables or Disables the use of count min sketch / heavy hitters for approximate frequency counts.
+      * cms_confidence (float) Defines the number of hashes used in CMS.
+        eg. confidence = 1 - failure probability, default 0.95
+      * cms_relative_error: (float) Defines the number of buckets used in CMS,
+            default 0.01
+      * cms_max_num_heavy_hitters (int) The value used to define
+        the threshold for minimum frequency required by a category to be counted
```
Comment (taylorfturner) [inline docs/source/profiler.rst:770]: indent these for clarity where they fall?

File: docs/source/profiler.rst [hunk 2]
```diff
@@ -761,6 +761,13 @@ Below is an breakdown of all the options.
       * top_k_categories - (int) Number of categories to be displayed when reporting
       * max_sample_size_to_check_stop_condition - (int) The maximum sample size before categorical stop conditions are checked
       * stop_condition_unique_value_ratio - (float) The highest ratio of unique values to dataset size that is to be considered a categorical type
+      * cms - (Boolean) Enables or Disables the use of count min sketch / heavy hitters for approximate frequency counts.
```
Comment (taylorfturner) [inline docs/source/profiler.rst:764]: oh interesting... I was expecting `CMS` to be its own options class .... 🤔

File: docs/source/profiler.rst (hunk 1 above)
Comment (taylorfturner) [reply docs/source/profiler.rst:770]: maybe don't indent if its all attributes under `CategoricalOptions`

File: docs/source/profiler.rst [hunk 3]
```diff
@@ -761,6 +761,13 @@ Below is an breakdown of all the options.
...
+      * cms_confidence (float) Defines the number of hashes used in CMS.
+        eg. confidence = 1 - failure probability, default 0.95
```
Comment (taylorfturner) [inline docs/source/profiler.rst:766]: does this need a tab for rendering?

File: docs/source/profiler.rst [hunk 4]
```diff
@@ -761,6 +761,13 @@ Below is an breakdown of all the options.
...
+      * cms_relative_error: (float) Defines the number of buckets used in CMS,
+            default 0.01
+      * cms_max_num_heavy_hitters (int) The value used to define
```
Comment (taylorfturner) [inline docs/source/profiler.rst:769]:
```suggestion
      * cms_max_num_heavy_hitters - (int) The value used to define
```

File: docs/source/profiler.rst [hunk 5]
```diff
@@ -761,6 +761,13 @@ Below is an breakdown of all the options.
       * top_k_categories - (int) Number of categories to be displayed when reporting
       * max_sample_size_to_check_stop_condition - (int) The maximum sample size before categorical stop conditions are checked
       * stop_condition_unique_value_ratio - (float) The highest ratio of unique values to dataset size that is to be considered a categorical type
+      * cms - (Boolean) Enables or Disables the use of count min sketch / heavy hitters for approximate frequency counts.
+      * cms_confidence (float) Defines the number of hashes used in CMS.
```
Comment (taylorfturner) [inline docs/source/profiler.rst:765]:
```suggestion
      * cms_confidence - (float) Defines the number of hashes used in CMS.
```

File: docs/source/profiler.rst [hunk 6]
```diff
@@ -761,6 +761,13 @@ Below is an breakdown of all the options.
...
+        eg. confidence = 1 - failure probability, default 0.95
+      * cms_relative_error: (float) Defines the number of buckets used in CMS,
```
Comment (taylorfturner) [inline docs/source/profiler.rst:767]:
```suggestion
      * cms_relative_error - (float) Defines the number of buckets used in CMS,
```

**Review by taylorfturner — CHANGES_REQUESTED** [review]

File: docs/source/profiler.rst (hunk 1 above)
Comment (tyfarnan) [reply docs/source/profiler.rst:770]: following pattern of pre-existing options at the same level

File: docs/source/profiler.rst (hunk 2 above)
Comment (tyfarnan) [reply docs/source/profiler.rst:764]: yup it's a method for doing approximate frequency counts. Only intended for the categorical profile for now.

File: docs/source/profiler.rst (hunk 3 above)
Comment (tyfarnan) [reply docs/source/profiler.rst:766]: I don' tthink so?

File: docs/source/profiler.rst [hunk 7]
```diff
@@ -761,6 +761,13 @@ Below is an breakdown of all the options.
       * top_k_categories - (int) Number of categories to be displayed when reporting
       * max_sample_size_to_check_stop_condition - (int) The maximum sample size before categorical stop conditions are checked
       * stop_condition_unique_value_ratio - (float) The highest ratio of unique values to dataset size that is to be considered a categorical type
+      * cms - (Boolean) Enables or Disables the use of count min sketch / heavy hitters for approximate frequency counts.
+      * cms_confidence - (float) Defines the number of hashes used in CMS.
+        eg. confidence = 1 - failure probability, default 0.95
```
Comment (taylorfturner) [inline docs/source/profiler.rst:766]:
```suggestion
      * cms_confidence - (float) Defines the number of hashes used in CMS. E.G. confidence = 1 - failure probability, default 0.95
```

File: docs/source/profiler.rst [hunk 8]
```diff
@@ -761,6 +761,13 @@ Below is an breakdown of all the options.
...
+      * cms_relative_error - (float) Defines the number of buckets used in CMS,
+            default 0.01
```
Comment (taylorfturner) [inline docs/source/profiler.rst:768]:
```suggestion
      * cms_relative_error - (float) Defines the number of buckets used in CMS, default 0.01
```

**Review by taylorfturner — APPROVED** [review]
LGTM

**Pull request merged**